### PR TITLE
Fix "Empty Count" error of swiftlint

### PR DESCRIPTION
**Pull request by limsangjin12**
I failed to build `HeroExamples` project with "Empty Count" error, and fixed it.

### CODE REVIEW

**Comment by limsangjin12** [comment]
I've pushed 1 more commit to fix all swiftlint warnings :)

File: Sources/HeroDebugView.swift [hunk 1]
```diff
@@ -23,7 +23,7 @@
 import UIKit
 
 #if os(iOS)
-protocol HeroDebugViewDelegate {
+protocol HeroDebugViewDelegate: class {
```
Comment (lkzhaoSC) [inline Sources/HeroDebugView.swift]: Why is this needed?

Comment (limsangjin12) [inline Sources/HeroDebugView.swift]: @lkzhaoSC Because of this (https://github.com/realm/SwiftLint/blob/master/Source/SwiftLintFramework/Rules/ClassDelegateProtocolRule.swift#L20).

**Comment by lkzhao** [comment]
Ahh great! Thanks.
